### PR TITLE
API-7592: activate -> resume chat

### PR DESCRIPTION
**Pull request by lpmichalik**
start/resume_chat: `active`

## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7592-resume-and-offline-thread)
- [Jira](https://livechatinc.atlassian.net/browse/API-7592)

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

File: content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx [hunk 1]
```diff
@@ -1061,6 +1061,7 @@ When `chat.users` is defined, one of following scopes is required:
 | `chat.thread`            | No       | `object`   |                                                                                                          |
 | `chat.thread.events`     | No       | `array`    | The list of initial chat events                                                                          |
 | `chat.thread.properties` | No       | `object`   |                                                                                                          |
+| `active`                 | No       | `bool`     | When set to `false` creates an inactive thread; default: `true`.                                         |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx:1064]:
```suggestion
| `active`                 | No       | `bool`     | When set to `false`, creates an inactive thread; default: `true`.                                         |
```

File: content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx [hunk 2]
```diff
@@ -1188,22 +1189,23 @@ When `chat.users` is defined, one of following scopes is required:
 | `chat.thread`            | No       | `object`   |                                                                                                          |
 | `chat.thread.events`     | No       | `array`    | Initial chat events array                                                                                |
 | `chat.thread.properties` | No       | `object`   | Initial thread properties                                                                                |
+| `active`                 | No       | `bool`     | When set to false, creates new inactive thread. Default `true`.                                          |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx:1192]:
```suggestion
| `active`                 | No       | `bool`     | When set to `false`, creates an inactive thread. Default `true`.                                          |
```

File: content/messaging/customer-chat-api/changelog/index.mdx [hunk 3]
```diff
@@ -14,6 +14,8 @@ This document is the record of all the changes in the **Customer Chat API**, Web
 ### Chats
 
 - In the **Deactivate Chat** ([Web](/messaging/customer-chat-api/v3.3/#deactivate-chat) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#deactivate-chat)) method, the `chat_id` parameter was renamed to `id`.
+- The **Activate Chat** has been renamed to **Resume Chat** ([Web](/messaging/customer-chat-api/v3.3/#resume-chat) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#resume-chat)).
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/changelog/index.mdx:17]:
```suggestion
- The **Activate Chat** method was renamed to **Resume Chat** ([Web](/messaging/customer-chat-api/v3.3/#resume-chat) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#resume-chat)).
```

File: content/messaging/customer-chat-api/changelog/index.mdx [hunk 4]
```diff
@@ -14,6 +14,8 @@ This document is the record of all the changes in the **Customer Chat API**, Web
...
+- The **Start Chat** ([Web](/messaging/agent-chat-api/v3.3/#start-chat) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#start-chat)) and **Resume Chat** ([Web](/messaging/customer-chat-api/v3.3/#resume-chat) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#resume-chat)) methods got a new optional parameter `active`.
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/changelog/index.mdx:18]:
```suggestion
- The **Start Chat** ([Web](/messaging/agent-chat-api/v3.3/#start-chat) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#start-chat)) and **Resume Chat** ([Web](/messaging/customer-chat-api/v3.3/#resume-chat) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#resume-chat)) methods have a new optional parameter, `active`.
```

File: content/messaging/customer-chat-api/v3.3/index.mdx [hunk 5]
```diff
@@ -804,6 +804,7 @@ Starts a chat.
 | `chat.thread`            | No       | `object`  |                                                                             |
 | `chat.thread.events`     | No       | `array`   | Initial chat events array                                                   |
 | `chat.thread.properties` | No       | `object`  | Initial thread properties                                                   |
+| `active`                 | No       | `bool`    | When set to `false` creates an inactive thread; default: `true`.            |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.3/index.mdx:807]:
```suggestion
| `active`                 | No       | `bool`    | When set to `false`, creates an inactive thread; default: `true`.            |
```

File: content/messaging/customer-chat-api/v3.3/index.mdx [hunk 6]
```diff
@@ -849,37 +850,38 @@ curl -X POST \
...
 | `chat`                   | Yes      | `object` |                                                                            |
-| `chat.id`                | Yes      | `string` | ID of the chat that will be activated.                                     |
+| `chat.id`                | Yes      | `string` | ID of the chat that will be resumed.                                       |
 | `chat.access`            | No       | `object` | Chat access to set; default: all agents.                                   |
 | `chat.properties`        | No       | `object` | Initial chat properties                                                    |
 | `chat.thread`            | No       | `object` |                                                                            |
 | `chat.thread.events`     | No       | `array`  | Initial chat events array                                                  |
 | `chat.thread.properties` | No       | `object` | Initial thread properties                                                  |
+| `active`                 | No       | `bool`   | When set to `false` creates an inactive thread; default: `true`.           |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.3/index.mdx:876]:
```suggestion
| `active`                 | No       | `bool`   | When set to `false`, creates an inactive thread; default: `true`.           |
```

File: content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx [hunk 7]
```diff
@@ -859,6 +859,7 @@ Starts a chat.
 | `chat.thread`            | No       | `object`  |                                                                             |
 | `chat.thread.events`     | No       | `array`   | Initial chat events array                                                   |
 | `chat.thread.properties` | No       | `object`  | Initial thread properties                                                   |
+| `active`                 | No       | `bool`    | When set to `false` creates an inactive thread; default: `true`.            |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx:862]:
```suggestion
| `active`                 | No       | `bool`    | When set to `false`, creates an inactive thread; default: `true`.            |
```

File: content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx [hunk 8]
```diff
@@ -947,37 +948,38 @@ Starts a chat.
...
 | `chat`                   | Yes      | `object` |                                                                            |
-| `chat.id`                | Yes      | `string` | The ID of the chat that will be activated.                                 |
+| `chat.id`                | Yes      | `string` | The ID of the chat that will be resumed.                                   |
 | `chat.access`            | No       | `object` | Chat access to set, default: all agents.                                   |
 | `chat.properties`        | No       | `object` | Initial chat properties                                                    |
 | `chat.thread`            | No       | `object` |                                                                            |
 | `chat.thread.events`     | No       | `array`  | Initial chat events array                                                  |
 | `chat.thread.properties` | No       | `object` | Initial thread properties                                                  |
+| `active`                 | No       | `bool`   | When set to `false` creates an inactive thread; default: `true`.           |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx:974]:
```suggestion
| `active`                 | No       | `bool`   | When set to `false`, creates an inactive thread; default: `true`.           |
```

File: content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx [hunk 9]
```diff
@@ -1018,12 +1020,12 @@ Used to restart an archived chat.
 
 </Section>
 
-<!-- > **`activate_chat`** sample **request** with optional params
+<!-- > **`resume_chat`** sample **request** with optional params
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx:1023]: This is some legacy stuff that doesn't need to be updated. Feel free to delete it next time you come across it.

File: content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx [hunk 10]
```diff
@@ -1214,12 +1216,12 @@ When `chat.users` is defined, one of following scopes is required:
 
 </CodeSample>
 
-<!-- > **`activate_chat`** sample **request** with optional params
+<!-- > **`resume_chat`** sample **request** with optional params
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx:1219]: This is some legacy stuff that doesn't need to be updated. Feel free to delete it next time you come across it.

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 11]
```diff
@@ -16,6 +16,8 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 - The **Deactivate Chat** ([Web](/messaging/agent-chat-api/v3.3/#deactivate-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#deactivate-chat)) method had the `chat_id` parameter renamed to `id`.
 - The **Follow Chat** ([Web](/messaging/agent-chat-api/v3.3/#follow-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#follow-chat)) method had the `chat_id` parameter renamed to `id`.
 - The **Unfollow Chat** ([Web](/messaging/agent-chat-api/v3.3/#unfollow-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#unfollow-chat)) method had the `chat_id` parameter renamed to `id`.
+- The **Activate Chat** method has been renamed to **Resume Chat** ([Web](/messaging/agent-chat-api/v3.3/#resume-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#resume-chat)).
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:19]:
```suggestion
- The **Activate Chat** method was renamed to **Resume Chat** ([Web](/messaging/agent-chat-api/v3.3/#resume-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#resume-chat)).
```

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/changelog/index.mdx]: I missed it, sorry.

**Review by ale-kacperczyk — APPROVED** [review]
LGTM 👍🏻

**Review by bkubiak — APPROVED** [review]

**Pull request merged**